closure/goog/dom/forms_test.js

```diff
@@ -417,6 +417,26 @@ function testSetValueInput() {
   assertEquals('', goog.dom.forms.getValue(el));
 }
 
+function testGetValueMeter() {
```

Comment:
the tests don't seem to be running in IE8 on Travis from what i can tell in the logs.

https://travis-ci.org/google/closure-library/builds/247947020?utm_source=github_status&utm_medium=notification

I'm seeing chrome, ff, ie9 and ie11

I don't have a copy of ie8 handy, and tbh i'm not even sure how to run the tests for closure locally.

I'm not sure how to reproduce the failure somewhere that i can see, and then fix it. Is this something you could help with @shicks ?